theblues/jem.py

```diff
@@ -47,22 +51,20 @@ def fetch_macaroon(self):
         return json.dumps(raw_macaroon)
 
     def get_users_models(self, macaroons):
-        """ Get the user's models from the JEM service.
+        """ Get the logged in user's models from the JEM service.
 
         @param macaroons The discharged JEM macaroons.
-
-        Return the json decoded list of environments.
+        @return The json decoded list of environments.
         """
         return make_request("{}env".format(self.url), macaroons=macaroons)
 
     def get_model(self, macaroons, user, name):
         """ Get a specified model.
 
         @param macaroons The discharged JEM macaroons.
-        @param user The username.
+        @param user The model's owner's username.
```

Comment:
The double possessives are hard to read.  Wouldn't "The model owner's username" say the same thing?

Or "The username of the model's owner"